webapp/README.MD

```diff
@@ -0,0 +1,34 @@
+# Running webapp Instructions:
+
+## For MAC OS/Linux:
+- Install pip3
+    ```sh
+    sudo apt-get install python3-pip
+    ```
+
+- Install virtualenv
+    ```sh
+    pip3 install virtualenv
+    ```
+
+- Change Directory
+    ```sh
+    cd <project-path>/devops-students/webapp
+    ```
+
+- Create Virtual Env
+    ```sh
+    virtualenv -p python3 venv
+    ```
+
+- Activate Virtual Env
+    ```sh
+    source venv/bin/activate
+    ```
+
+- Install Project Requirements
+    ```sh
+    pip3 install -r requirements.txt
+    ```
+
+## For Windows 10/11:
```

Comment:
We need to figure out the Windows 10/11 steps and add them in. Your first step will be installing Python3 and pip3 on windows.